### ISSUE TITLE
Needs a Menu accessible in game

### ISSUE THREAD

**Issue by mikeloeven** [moonlight-stream/moonlight-android#270]
I have tried both long press back and long press menu however the program seems to lack any form of context menu accessible from in game. additionally pressing back does not appear to return to the menu or end your current session. 

Additionally if a context menu is implemented it needs to have a function for showing the OSK when text put is needed and option to show floating mouse buttons allowing the user to easily access middle mouse and right click. 

Basically this applications needs pretty much all the input methods of a fully featured remote desktop client since many newer phones lack a hardware keyboard.

**Comment by cgutman** [comment]
I plan to add an overlay in the future. However, you can access right-click and keyboard functionality today.

Tapping a second finger while holding down one finger will right click (like the trackpad gesture). Tapping 3 fingers together will bring up the software keyboard.

**Comment by mikeloeven** [comment]
I didn't even know that functionality was implemented maybe you should have a tutorial pop-up display the first time it's run after installation
